gecko_client/info_api.go

```diff
@@ -0,0 +1,41 @@
+package gecko_client
+
+import (
+	"encoding/json"
+	"github.com/palantir/stacktrace"
+)
+
+const (
+	infoEndpoint = "ext/info"
+)
+
+type InfoApi struct {
+	rpcRequester jsonRpcRequester
+}
+
+func (api InfoApi) GetPeers() ([]Peer, error) {
+	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(infoEndpoint, "info.peers", make(map[string]interface{}))
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "Error making request")
+	}
+
+	var response GetPeersResponse
```

Comment:
This type looks like it's still coming from the admin API - can we make sure they're separate (since they're technically separate endpoints right now)?

Reply:
deleting this whole thing